src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/InsertOneTableIntegrationTest.java

```diff
@@ -1671,17 +1673,23 @@ void failOnVectorSizeMismatch() {
           .templated()
           .insertOne(
                   """
-                      { "id": "vectorValidBase64",
-                        "vector": {"$binary": "%s"}
-                      }
-                      """
+                              { "id": "vectorValidBase64",
+                                "vector": {"$binary": "%s"}
+                              }
+                              """
                   .formatted(Base64Util.encodeAsMimeBase64(floatsAsBytes)))
           .hasSingleApiError(
               DocumentException.Code.INVALID_COLUMN_VALUES,
               DocumentException.class,
               "vector(vector) - Cause: expected vector of length 3, got one with 2 elements");
     }
+  }
 
+  @Nested
+  @Order(12)
+  @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
Ordering ITs because one of these fails sporadically on CI. Hoping it is order-specific and adding ordering either reproduces the issue, or prevents it.